docs/installation.rst

```diff
@@ -10,3 +12,10 @@ Or, if you have virtualenvwrapper installed::
 
     $ mkvirtualenv durga
     $ pip install durga
+
+
+From Gitub
+===========
+You can install `durga` from source just as you would install any other Python package::
+
+    $ pip install git+https://github.com/transcode-de/durga.git
```

Comment:
Installing this way is not a good idea:

1. You have to have git installed.
2. This will clone the repository which takes some time but has no advantage.
3. You get the latest revision from the main branch.

The question is: Why should someone install the distribution not from PyPI?